rs_server/CADIP/api/cadu_download.py

```diff
@@ -231,6 +232,9 @@ def download(
             content={"started": "false"},
         )
 
+    # Reset status to not_started
+    db_product.not_started(db)
```

Comment:
This is useful for the demo if we download the product several times, to pass the status from done -> not_started, so when we download the second time and check the status, we are sure it is not_started and not done.